enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -162,107 +241,82 @@ spec:
   containers:
     ...
      volumeMounts:
-        - name: entitlement
-          mountPath: /etc/entitlement
+        - name: shared-item-name
+          mountPath: /path/to/shared-item-data
   volumes:
       - name: entitlement
         csi:
           driver: secretprovider.openshift.io
```

Comment:
update for projectedresources and the name of the projectedresource?

Reply:
yep will make consistent with prior yam in doc in next push